README.md

```diff
@@ -34,6 +34,7 @@ To install PyTorch/XLA nightly build in a new TPU VM:
 ```
 pip3 install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
 pip3 install https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-nightly-cp310-cp310-linux_x86_64.whl
+pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html
```

Comment:
You should be able to do `pip install 'torch_xla[tpu] @ https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-nightly-cp310-cp310-linux_x86_64.whl' -f https://storage.googleapis.com/libtpu-releases/index.html` 

The advantage here is that `pip` will fail if there's a dependency issue (reasonably likely since you hardcoded python 3.10 in the command) rather than reinstalling a non-nightly `torch_xla` from PyPI. You should also make a note to change the Python version if necessary.